lib/customerio/client.rb

```diff
@@ -48,44 +37,49 @@ def track(*args)
       if args.length == 1
         # Only passed in an event name, create an anonymous event
         event_name = args.first
+        raise ParamError.new("event_name must be a non-empty string") if is_empty?(event_name)
```

Comment:
This isn't what the golang library does.

Reply:
Dropped the case that handles single args and sends an anonymous event.